packages/config/config/devices/0x010f/fghc3-001.json

```diff
@@ -0,0 +1,27 @@
+{
+	"manufacturer": "Fibargroup",
+	"manufacturerId": "0x010f",
+	"label": "FGHC3-001",
+	"description": "Home Center 3",
```

Comment:
I didn't know this could be included as a secondary

Reply:
Looks like it: https://manuals.fibaro.com/document-category/hc3-configuration/